spec/twilio_txt_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'twilio_txt'
+
+describe Sms do
+
+  let!(:client) { double(:client) }
+  let(:subject) { Sms.new(client) }
+
+  it 'returns a message when called' do
+    chain = "account.messages.create"
+    allow(client).to receive_message_chain(chain) { 'output message' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.